src/lib/utils/env.js

```diff
@@ -0,0 +1,8 @@
+import config from '%src/config'
+import {
+	ENV_NAMES,
+} from '%src/constants/env'
```

Comment:
This format is totally fine but I will note that when we're only importing a single item, it's perfectly acceptable to use the one-line `import { ENV_NAMES } from '%src/constants/env'`. We only "enforce" multi-line imports when dealing with 2+ imported items, since it makes diffs so much more legible. (We would then convert to multi-line when importing a second item from this file, which would make a very obvious/legible diff; same when converting from multiple imports to a single one.)

Like I said, this is fine, but you can feel free to collapse it to a single line if you want, too.